base_layer/core/src/block.rs

```diff
@@ -30,17 +30,18 @@ use crate::{
     transaction_protocol::{build_challenge, TransactionMetadata},
     types::*,
 };
+use serde::{Deserialize, Serialize};
 use tari_crypto::{
     commitment::HomomorphicCommitmentFactory,
     keys::{PublicKey, SecretKey},
     range_proof::RangeProofService,
 };
-use tari_utilities::byte_array::ByteArray;
+use tari_utilities::byte_array::*;
```

Comment:
Generally I prefer not using wildcards, so it's easy to read exactly where imported objects comes from